beam/www/beam/components/Transfer.vue

```diff
@@ -34,6 +37,7 @@ const create = async () => {
 	if (data.name) {
 		stockEntryId.value = data.name
 	}
+	return { data }
```

Comment:
I think this should return the other values from the `insert` call as well (`exception`, `response`).